src/utils/enums.ts

```diff
@@ -1,13 +1,24 @@
-// eslint-disable-next-line no-shadow
+/* eslint no-shadow: 0 */
+/**
+ * Enum of UnitType
+ */
 export enum UnitType {
-  DIVISION = 'division',
-  BRIGADE = 'brigade',
-  BATTALION = 'battalion',
-  COMPANY = 'company'
+  COMPANY = 0,
+  BATTALION = 1,
+  BRIGADE = 2,
+  DIVISION = 3,
```

Comment:
Is there a reason to change the value from string to int?

Reply:
It will probably make it easier when we insert row to db if we keep it as string.

Reply:
I used the int to compare the hierarchy of UnitType when generate questions. We can have constant map to convert back and forth if we need that for DB

Reply:
Cool!